src/WinRT.Runtime/ComWrappersSupport.net5.cs

```diff
@@ -433,8 +432,13 @@ private static unsafe bool IsRuntimeImplementedRCW(Type objType)
             return isRcw;
         }
 
+        // This is used to hold the reference to the native value type object (IReference) until the actual value in it (boxed as an object) gets cleaned up by GC
+        // This is done to avoid pointer reuse until GC cleans up the boxed object
+        private static ConditionalWeakTable<object, IObjectReference> valueTypeRefHolder = new ConditionalWeakTable<object, IObjectReference>();

```

Comment:
This cache needs to live in the ComWrappers implementation itself to handle the Reference Tracker Runtime case that caused the last fix to fail.

Reply:
This is already in ComWrappers implementation - DefaultComWrappers. Do you mean something else?

Reply:
Sorry, I thought this was in the `ComWrappersSupport` class. GitHub's review preview is misleading.